sky_alert/endpoint.py

```diff
@@ -7,6 +14,27 @@
 def healthz():
     return 'OK'
 
-@app.get('/sunrise')
-def sunrise():
-    return '06:00'
\ No newline at end of file
+@app.get('/sundata')
+def sun_data(lat=0, lon=0):
+
+    headers = {
+        'Content-Type': 'application/json',
+    }
+
+    params = {
+        "lat": lat,
+        "lon": lon,
+        "appid": os.environ["OPENWEATHER_API_KEY"]
+    }
+    
+    res = requests.get(os.environ["OPENWEATHER_API_URL"], params=params, headers=headers)
+
+    sunrise = datetime.datetime.utcfromtimestamp(res.json()['current']['sunrise'])
+    sunset = datetime.datetime.utcfromtimestamp(res.json()['current']['sunset'])
+
+    sundata = SunData(sunrise, sunset)
```

Comment:
```suggestion
    sun_data = SunData(sunrise, sunset)
```
snake case for variable names, to not confuse with class names (camel case)